src/parameters.jl

```diff
@@ -20,18 +20,19 @@ value(x::NamedTuple) = map(value, x)
 value(x::Dict) = Dict(k => value(v) for (k, v) in x)
 
 """
-    positive(val::Real, transform::Bijector=Bijectors.Exp(), ε::Real = 1e-12)
+    positive(val::T, transform::Bijector=Bijectors.Exp(), ε=sqrt(eps(T))) where T<:Real
```

Comment:
Because we're changing the default tolerance this is technically a breaking change.